Dockerfile

```diff
@@ -8,13 +8,14 @@ COPY package.json package-lock.json tsconfig.json ./
 
 RUN --mount=type=cache,target=/root/.npm npm ci
 
-RUN npm run build
+RUN npm run build:production
 
 FROM node:22-alpine AS release
 
 COPY --from=builder /app/dist /app/dist
 COPY --from=builder /app/package.json /app/package.json
 COPY --from=builder /app/package-lock.json /app/package-lock.json
+COPY --from=builder /app/scripts/set-release-stage.js /app/scripts/set-release-stage.js
```

Comment:
you also need to add the script in the builder stage, adding something like:

`COPY scripts/set-release-stage.js ./scripts/set-release-stage.js`
 above in the builder stage should fix this copy as well